contrib/gitian/gitian-android.yml

```diff
@@ -0,0 +1,135 @@
+---
+name: "monero-android-0.15"
+enable_cache: true
+suites:
+- "bionic"
+architectures:
+- "amd64"
+packages:
+- "curl"
+- "gperf"
+- "gcc-7"
+- "g++-7"
+- "gcc"
+- "g++"
```

Comment:
Other config in this PR uses CLANG. Is this intended ?

Reply:
These are building anything that the build machine needs. clang is only for building target binaries, and comes from the Android NDK.